examples/http-sec/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	httpCmp, err := newHTTPComponent(kafkaBroker, kafkaTopic, "http://localhost:50000/kafka")
+	asyncComp, err := newAsyncKafkaProducer(kafkaBroker, kafkaTopic)
```

Comment:
renaming for more semantic meaning... 

I think it was httpCmp before as it was probably copied from the `http/main.go` file as a template.